bricks/basic/src/table/index.tsx

```diff
@@ -0,0 +1,1692 @@
+    Record<string, number>
+  >;
+
+  /**
+   * @detail Record<string,any>[]
+   * @description 勾选框变化，detail 中为所选的行数据
+   */
+  @event({ type: "select.update" }) accessor #selectUpdate!: EventEmitter<
+    Record<string, any>[]
+  >;
+
+  /**
+   * @detail Record<string,any>[]
+   * @description 勾选框变化，detail 中为所选的行key集合
+   */
+  @event({ type: "select.row.keys.update" })
+  accessor #selectRowKeysUpdate!: EventEmitter<string[]>;
+
+  /**
+   * @detail {sort:string;order:string|number}
+   * @description 排序变化，detail 中的 sort 为对应排序列的 key/dataIndex，order 为升序/降序
+   */
+  @event({ type: "sort.update", cancelable: true })
+  accessor #sortUpdate!: EventEmitter<{
+    sort: string;
+    order: string | number;
+  }>;
+
+  /**
+   * @detail {expanded:boolean;record:Record<string,any>}
+   * @description 点击展开图标时触发的事件，事件详情中`expanded`为是否展开，`record`被点击的行信息
+   */
+  @event({ type: "row.expand" }) accessor #rowExpand!: EventEmitter<{
+    expanded: boolean;
+    record: Record<string, any>;
+  }>;
+
+  /**
+   * @detail {expandedRows:string[]| number[]}
+   * @description 展开的行变化时触发的事件，事件详情为当前展开的所有行的`rowKey`集合
+   */
+  @event({ type: "expand.rows.change" })
+  accessor #expandRowsChange!: EventEmitter<{
+    expandedRows: React.Key[];
+  }>;
+
+  /**
+   * @detail {data:Record<string,any>[]}
+   * @description 表格行拖拽结束发生的事件，事件详情为拖拽后重新排序的所有行数据
+   */
+  @event({ type: "row.drag" }) accessor #rowDrag!: EventEmitter<{
+    data: Record<string, any>[];
+  }>;
+
+  /**
+   * @kind CustomColumn[]
+   * @required false
+   * @default -
+   * @description 扩展自 ant-design 的 Column 相关配置项,具体查阅：[Column](https://ant.design/components/table-cn/#Column)
+   * @group basic
+   */
+  set columns(value: CustomColumn[]) {
+    this._columns = value;
+    this._render();
+  }
+  get columns(): CustomColumn[] {
+    return this._columns as CustomColumn[];
+  }
+
+  /**
+   * @kind any[]
+   * @required false
+   * @default -
+   * @description 数据源，通过 useResolves 从后台接口获取或者直接在 storyboard 中配置
+   * @group basic
+   */
+  set dataSource(value: Record<string, any>[]) {
+    this._isInSelect = false;
+    this._originalDataSource = value;
+    this._dataSource = cloneDeep(
+      this._fields.dataSource ? get(value, this._fields.dataSource) : value
+    );
+    // 前端搜索需要保留一个干净的数据
+    this._pureSource = cloneDeep(this._dataSource);
+    if (this.stripEmptyExpandableChildren) {
+      const columnName =
+        this.configProps?.expandable?.childrenColumnName || "children";
+      stripEmptyExpandableChildrenByName(columnName, this._dataSource);
+    }
+    this.#total = get(value, this._fields.total as string);
+    this.page = get(value, "page") ?? this.page ?? 1;
+    this.pageSize =
+      get(value, "page_size") ?? get(value, "pageSize") ?? this.pageSize ?? 10;
+    this._render();
+  }
+
+  /**
+   * @required false
+   * @default true
+   * @description 是否显示外层卡片
+   * @group basic
+   */
+  @property({
+    attribute: false,
```

Comment:
V3 支持带默认值的 prop <--> attr 映射

```suggestion
    type: Boolean,
```